windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md

```diff
@@ -225,28 +225,28 @@ The controller shall promptly generate a Command Completed event in response to
 
 #### Command_parameters
 
-Subcommand_opcode (one octet):
+Subcommand_opcode (1 octet):
 
 | Value | Parameter description |
 |--|--|
 | 0x02 | The subcommand opcode for [HCI_VS_MSFT_Cancel_Monitor_Rssi][ref_HCI_VS_MSFT_Cancel_Monitor_Rssi]. |
 
-Connection_Handle (one octet):
+Connection_Handle (2 octets):
```

Comment:
Note that this changes both the number and the plurality. This always should have been "2 octets" (not "1 octet").